include/smac_planner/utils.hpp

```diff
@@ -81,6 +81,27 @@ class Utils {
       return (dot < 0);
     }
 
+
+  static geometry_msgs::Pose getPoseAtDistanceAlongHeading(const geometry_msgs::Pose& pose, const float& distance){
+    geometry_msgs::Pose output_pose;
```

Comment:
S:
```suggestion
    geometry_msgs::Pose output_pose = pose;
```
then can remove copy of z and orientation